### PR TITLE
1cake

### PR DESCRIPTION
**Pull request by catalinac3**
closes #6 
the data from one cake displayed in simple html :)
- cake label 
- img
- list of ingredients
-a button with a link to the recipe

I don't think the API free plan provides the recipe itself, that is why I include the link

### CODE REVIEW

File: app.js
Comment (marsamak) [inline app.js]: I would use const and put ; at the end of the line:)

File: app.js
Comment (marsamak) [inline app.js]: I would use const keyword. So actually, when you a not going to reassign your variable, it's better to declare it as const.

**Review by marsamak — CHANGES_REQUESTED** [review]

**Comment by marsamak** [comment]
It looks good to me apart from these minor issues with const. I don't know, if I'm doing it right, never done a code review from my handy using this UI provided by github:) You could perform this little refactoring and merge everything into main branch.

**Comment by catalinac3** [comment]
I don't really get the option to merge, normally you wait for re-approval when changes are suggested, but now the pull request is closed on my side, so I reopened, you can click on approve and I can merge it  :), same with the last one.
I think we are going to learn a lot from this Surprise :) :D , you are a natural at reviewing !!!, just the detail of not closing the pull request, so I can see that I have work to do on my github

Actually once is open I can merge it...

**Pull request merged**
